CMake/External_GDAL.cmake

```diff
@@ -118,6 +130,15 @@ else()
     set(_GDAL_ARGS_XML2 "--with-xml2=${LIBXML2_ROOT}/bin/xml2-config")
   endif()
 
+  if(fletch_ENABLE_GEOS)
+    list(APPEND _GDAL_DEPENDS GEOS)
+    set( _GDAL_GEOS_ARGS "--with-geos=${GEOS_ROOT}")
+    message(STATUS "GEOS_ROOT = ${GEOS_ROOT}")
+  else()
+    # TODO / FIXME: allow use of system geos? What is the best way to add that option?
+    set(_GDAL_GEOS_ARGS --with-geos=no)
+  endif()
+
```

Comment:
Since we provide a Windows build of GEOS, we should add the appropriate version of this block up there too.

Reply:
I added what I believe to be the appropriate code to the win32 section (I was following patterns of the other depends libs; I'm not certain that what I did was correct). 

I noticed that the other win32 depends libs had their library name explicitly written out: e.g. `GEOTIFF_LIB=${fletch_BUILD_INSTALL_PREFIX}/lib/geotiff_i.lib`. However, all the external_*.cmake files seem to define variables like `libgeotiff_LIBRARY`, `TIFF_LIBRARY`, and `PNG_LIBRARY` that have the same value, so I switched everything to use those variables instead. I think that's a positive change, but let me know if there is some reason that we shouldn't do that that I'm not considering.